docs/MockFunctions.md

```diff
@@ -106,7 +106,7 @@ const result = [11, 12].filter(num => filterTestFn(num));
 console.log(result);
 // > [11]
 console.log(filterTestFn.mock.calls[0][0]); // 11
-console.log(filterTestFn.mock.calls[1][0]); // 12
+console.log(filterTestFn.mock.calls[0][1]); // 12
```

Comment:
no, it's called 2 times (first index access) with one argument (second index access).